chromedp/click/main.go

```diff
@@ -111,6 +113,23 @@ func run(ctx context.Context, args []string, stdout, stderr io.Writer) error {
 		return fmt.Errorf("click: %w", err)
 	}
 
+	// chromedp.WaitNewTarget is a higher level abstraction for this, but we
+	// need to start emiting Target events, specifically:
+	// https://chromedevtools.github.io/devtools-protocol/tot/Target/#event-targetCreated
+	// https://chromedevtools.github.io/devtools-protocol/tot/Target/#event-targetInfoChanged
+	clickComplete := make(chan struct{}, 1)
```

Comment:
you don't need to set the channel size to `1`. It's done by default by Go.

```suggestion
	clickComplete := make(chan struct{})
```